app/webpacker/css/admin_v3/all.scss

```diff
@@ -52,6 +52,7 @@
 @import "../admin/components/alert-box";
 @import "../admin/components/alert_row";
 @import "../admin/components/buttons";
```

Comment:
I think it would be best to stick to the rule of replacing the old file entirely

```suggestion
```

Reply:
Yep, I forget. Thanks.